tests/register.spec.coffee

```diff
@@ -18,17 +18,15 @@ if IS_BROWSER
 	realFetchModule = require('fetch-ponyfill')({ Promise })
 	_.assign(global, _.pick(realFetchModule, 'Headers', 'Request', 'Response'))
 else
-	settings = require('resin-settings-client')
-	dataDirectory = settings.get('dataDirectory')
+	temp = require('temp').track()
+	dataDirectory = temp.mkdirSync()
```

Comment:
@emirotin This change is worth noting. Not sure whether the original code was mine or came from one of your changes, but it broke on my machine today, because I had a local token, but it was expired, so resin-request tried to update it. If we use an empty temp dir it's reliably always fine everywhere. We should watch out for that elsewhere, and do this instead, so the tests stay stable for everybody.